arch-bootstrap.sh

```diff
@@ -116,6 +116,8 @@ configure_minimal_system() {
   touch "$DEST/etc/group"
   echo "bootstrap" > "$DEST/etc/hostname"
   
+  rm -rf $DEST/etc/mtab
```

Comment:
Deleting it seems less safe then ideal. Is there a reason you don't just move the file to `/etc/mtab.arch_bootstrap_backup`?